docs/02-Common-Architecture-Concepts/02-learning-goals.adoc

```diff
@@ -7,7 +7,7 @@
 Softwarearchitekt:innen kennen Muster moderner Architekturen für verteilte Anwendungen (speziell Microservices, Self-contained Systems) und verstehen die Gründe, die zu diesen Lösungen führen.
 
 * Monolith
-* Microsoervices
+* Microservices
 * Self Contained Systems
```

Comment:
Wollen wir das dann auch noch in `Self-contained Systems` umschreiben? Deutsch wie Englisch? 🙃